libsession/src/main/java/org/session/libsession/snode/SnodeAPI.kt

```diff
@@ -126,7 +137,8 @@ object SnodeAPI {
         useOnionRequests -> OnionRequestAPI.sendOnionRequest(method, parameters, snode, version, publicKey).map {
             JsonUtil.fromJson(it.body ?: throw Error.Generic, Map::class.java)
         }
-        else -> task {
+
+        else -> GlobalScope.asyncPromise {
```

Comment:
You created a scope line 106, can't you use it here?

Reply:
You are right. Replaced.

Reply:
There's one left line 616